src/renderer/components/dialogs/ViewProfile/styles.module.scss

```diff
@@ -6,4 +6,5 @@
 .viewProfileDialogBody {
   display: flex;
   flex-direction: column;
+  padding-bottom: 0.6px;
```

Comment:
zero padding resulted in a second scrollbar, I suspect the auto sizer rounds up the (ceil) the height it gets somewhere so it takes more than there actually is.